src/TypeScript/EslintBridgeClient/Contract/TSConfigResponse.cs

```diff
@@ -0,0 +1,46 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2021 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace SonarLint.VisualStudio.TypeScript.EslintBridgeClient.Contract
+{
+    internal class TSConfigResponse
+    {
+        /// <summary>
+        /// File paths to source files referenced by the tsconfig file
+        /// </summary>
+        [JsonProperty("files")]
+        public IEnumerable<string> Files { get; set; }
+
+        /// <summary>
+        /// File paths of referenced projects that should be resolved by calling "tsconfig-files"
```

Comment:
I'm not sure I understand this comment. Will we ever need to use this property? Or do we only care about "Files"?

Reply:
I was going to exclude the property, but I had a look at the Java code and it is used. The Java code parses the response and calls `tsconfig-files` for each value in `ProjectReferences` (which suggests that a "project reference" is another tsconfig file).